frontend/src/scenes/sessions/sessionsTableLogic.ts

```diff
@@ -10,6 +10,12 @@ import { sessionsFiltersLogic } from 'scenes/sessions/filters/sessionsFiltersLog
 
 type SessionRecordingId = string
 
+enum ExpandState {
+    expanded,
```

Comment:
Other enums are capitalized throughout the codebase - worth keeping it consistent.